shapeshift/web
# feat(idle): disable deposit on paused pools

## Description

Blocked by https://github.com/shapeshift/lib/pull/1226

This PR consumes the new exposed `active` property from investor-idle, and disables depositing on the Idle pools that aren't currently active.

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk



Low, as long as upstream data is safu we're safu

## Testing



- Go to Idle opportunities by provider
- Ensure that opportunities that are currently disabled upstream (can be cross-checked against Idle's UI) have their deposit disabled
- Other opportunities should still allow depositing

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)
